docs/ug/customizingReports.md

```diff
@@ -57,3 +57,5 @@ To ensure that their PRs are correct, you can use [Netlify _deploy previews_](ht
 In both instances, it is **necessary to commit any changes** for them to be detected by RepoSense.
 
 </box>
+
+3\. Add a git `.mailmap` file at the toplevel of the repository, specifying mapped authors/commiters and/or e-mail addresses as per [gitmailmap documentation](https://git-scm.com/docs/gitmailmap). Any mappings specified here will be applied by git before all other RepoSense configurations.
```

Comment:
It might be good if we can do a short elaboration on the benefits here, i.e., why use mailmap